packages/react/ds/src/hint-text/hint-text.tsx

```diff
@@ -9,7 +9,7 @@ export enum HintSize {
 // Extend `React.InputHTMLAttributes<HTMLInputElement>` so that
 // the component can accept all the standard attributes and events that an `<input>` element can handle.
 export type HintTextProps = React.HTMLAttributes<HTMLInputElement> & {
-  text: string;
+  text: ReactNode;
```

Comment:
Change this to accept things like: `<strong>Bold text</strong>` like the macro does.